### PR TITLE
Add Theme Page to Directory

### PR DESCRIPTION
**Pull request by EliteStarServices**
Added Theme Install Page
Minor Changes to Menus (uniformity)
Issue - Themes not activating

### CODE REVIEW

**Comment by xxsimoxx** [comment]
@EliteStarServices I've removed the menu renaming as the menu contains all plugins, not only WP's ones.

**Comment by EliteStarServices** [comment]
I shared the css/js thinking the existing files could be modified, then split into separate entities.

**Comment by xxsimoxx** [comment]
I've separated them early because I think they will differ very soon as Tim will open a PR for modals.

**Comment by EliteStarServices** [comment]
Sounds good, I did not know which way would end up being less work.

File: classes/PluginInstall.class.php [hunk 1]
```diff
@@ -45,7 +45,7 @@ public function create_menu()
 		$this->page = add_submenu_page(
 			'plugins.php',
 			esc_html__('Install ClassicPress plugins', 'classicpress-directory-integration'),
```
Comment (KTS915) [inline classes/PluginInstall.class.php:47]: Plugins here should also be capitalized:
```
esc_html__(' Install ClassicPress Plugins',
```

File: classes/PluginInstall.class.php [hunk 2]
```diff
@@ -62,13 +62,13 @@ public function rename_menu() {
...
-			$submenu['plugins.php'][$key][0] = esc_html__('Install WP plugins', 'classicpress-directory-integration'); // phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
+			$submenu['plugins.php'][$key][0] = esc_html__('Install WP Plugins', 'classicpress-directory-integration'); // phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
 		}
 	}
 
 	// Get all installed ClassicPress plugin
 	// This function is different from the one in PluginUpdate class
-	// and considers a plugin from the dir not only if if has UpdateURI
+	// and considers a plugin from the dir not only if it has UpdateURI
 	// but also if it have RequiresCP.
```
Comment (KTS915) [inline classes/PluginInstall.class.php:72]: This should be:
```
// but also if it has RequiresCP.
```

**Review by KTS915 — COMMENTED** [review]
Two typos to correct; otherwise looks good.

File: classes/PluginInstall.class.php (hunk 1 above)
Comment (EliteStarServices) [reply classes/PluginInstall.class.php:47]: I thought I fixed the lowercase Plugin.

File: classes/PluginInstall.class.php (hunk 2 above)
Comment (EliteStarServices) [reply classes/PluginInstall.class.php:72]: Missed that one.

**Comment by EliteStarServices** [comment]
Looks good to me then, I am sure we can knock any other dents out as we go.

File: classes/PluginInstall.class.php (hunk 2 above)
Comment (xxsimoxx) [reply classes/PluginInstall.class.php:72]: Done!

File: classes/PluginInstall.class.php (hunk 1 above)
Comment (xxsimoxx) [reply classes/PluginInstall.class.php]: Done!

File: classes/PluginInstall.class.php [hunk 3]
```diff
@@ -68,7 +68,7 @@ public function rename_menu() {
 
 	// Get all installed ClassicPress plugin
 	// This function is different from the one in PluginUpdate class
-	// and considers a plugin from the dir not only if it has UpdateURI
+	// and considers a plugin from the dir not only if it has RequiresCP
```
Comment (KTS915) [inline classes/PluginInstall.class.php:71]: UpdateURI

File: classes/PluginInstall.class.php [hunk 4]
```diff
@@ -68,7 +68,7 @@ public function rename_menu() {
...
 	// but also if it have RequiresCP.
```
Comment (KTS915) [inline classes/PluginInstall.class.php:72]: // but also if it has RequiresCP.

**Review by KTS915 — COMMENTED** [review]
The first one is fine; in the second one, you have changed the wrong line.

File: classes/PluginInstall.class.php (hunk 4 above)
Comment (KTS915) [reply classes/PluginInstall.class.php:72]: It's line 72 that needs changing; line 71 should be as it was with `UpdateURI`

Comment (xxsimoxx) [reply classes/PluginInstall.class.php]: Should be fixed.

File: classes/PluginInstall.class.php (hunk 3 above)
Comment (xxsimoxx) [reply classes/PluginInstall.class.php]: Should be fixed.

**Review by KTS915 — APPROVED** [review]

**Review by xxsimoxx — APPROVED** [review]

**Pull request merged**